tests/unit/Tax/Helpers/TaxCest.php

```diff
@@ -63,36 +59,30 @@ public function _before(UnitTester $I)
         Cart::addProduct($product1->id, [
             '_token'     => session('_token'),
             'product_id' => $product1->id,
-            'quantity'   => self::PRODUCT1_QTY,
+            'quantity'   => 11,
         ]);
 
         Cart::addProduct($product2->id, [
             '_token'     => session('_token'),
             'product_id' => $product2->id,
-            'quantity'   => self::PRODUCT2_QTY,
+            'quantity'   => 7,
         ]);
 
-        // rounded by precision of 2 because this are sums of corresponding tax categories
-        $expectedTaxAmount1 = round(
-            round(self::PRODUCT1_QTY * $product1->price, self::CART_TOTAL_PRECISION)
-            * $tax1->tax_rate / 100,
-            self::CART_TOTAL_PRECISION
-        );
-
-        $expectedTaxAmount2 = round(
-            round(self::PRODUCT2_QTY * $product2->price, self::CART_TOTAL_PRECISION)
-            * $tax2->tax_rate / 100,
-            self::CART_TOTAL_PRECISION
-        );
 
         $this->scenario = [
-            'cart'             => Cart::getCart(),
+            'object'           => Cart::getCart(),
             'expectedTaxRates' => [
-                (string)round((float)$tax1->tax_rate, 4) => $expectedTaxAmount1,
-                (string)round((float)$tax2->tax_rate, 4) => $expectedTaxAmount2,
+                (string)round((float)$tax1->tax_rate, 4)
+                => round(11 * $product1->price * $tax1->tax_rate / 100, 2),
```

Comment:
@Ashish503-web 
Let's suppose someone wants to change the price of product 1 and 2 in this test.

What will happen if your refactoring of this test will be merged?